### ISSUE TITLE
First execution error: Configuration property "zuix" is not defined

### ISSUE THREAD

**Issue by slaivyn** [zuixjs/zuix#18]
Hi,
I wanted to test zuix so I ran the following command: `npx zuix new my-appname`.
And I got this result:

```
WARNING: No configurations found in configuration directory:/home/sylvain/projects/distorsion/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.
/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/config/lib/config.js:179
    throw new Error('Configuration property "' + property + '" is not defined');
    ^

Error: Configuration property "zuix" is not defined
    at Config.get (/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/config/lib/config.js:179:11)
    at Object.<anonymous> (/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/zuix/commands/generate.js:36:25)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1067:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/zuix/zuix.js:33:18)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
```
Node version: `v16.19.0`

**Comment by slaivyn** [comment]
Is there another way to start a new app from one of the full featured templates?

Do you need more details to troubleshoot this bug?

**Comment by genemars** [comment]
Would you please provide the full terminal output?
I just tried and no errors so far:
```
$ npx zuix new my-webapp
- created folder "my-webapp"
- downloading web-starter
- extracting
- copying files
- installing packages

added 752 packages, and audited 755 packages in 11s

99 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Done!

$ cd my-webapp/

$ npx zuix start

> my-webapp@1.0.0 start
> npx @11ty/eleventy --serve

- created folder "docs/js/codemirror"
[11ty] Writing docs/manifest.json from ./source/manifest.json.liquid
[11ty] Writing docs/robots.txt from ./source/robots.njk
[11ty] Writing docs/search-index.json from ./source/search-index.json.njk
[11ty] Writing docs/sitemap.xml from ./source/sitemap.njk
[11ty] Writing docs/editor/index.html from ./source/editor/index.liquid
[11ty] Writing docs/feed/rss.xml from ./source/feed/rss.njk
[11ty] Writing docs/search/index.html from ./source/search.liquid
[11ty] Writing docs/content/docs/index.html from ./source/content/docs/index.liquid
[11ty] Writing docs/index.html from ./source/index.liquid
[11ty] Writing docs/content/docs/building-and-publishing/index.html from ./source/content/docs/building-and-publishing/index.md (liquid)
[11ty] Writing docs/content/docs/liquid-tags/index.html from ./source/content/docs/liquid-tags/index.md (liquid)
[11ty] Writing docs/content/docs/page-editing-and-options/index.html from ./source/content/docs/page-editing-and-options/index.md (liquid)
[11ty] Writing docs/content/docs/getting-started/index.html from ./source/content/docs/getting-started/index.md (liquid)

15:38:32.299 I editor/index.html
15:38:32.413 I  = skipped (same as input)
             | 
15:38:32.415 I search/index.html
15:38:32.565 I  = skipped (same as input)
             | 
15:38:32.567 I content/docs/index.html
15:38:32.654 I  = skipped (same as input)
             | 
15:38:32.656 I index.html
15:38:32.787 I  = skipped (same as input)
             | 
15:38:32.788 I content/docs/building-and-publishing/index.html
15:38:32.881 I  = skipped (same as input)
             | 
15:38:32.882 I content/docs/liquid-tags/index.html
15:38:33.035 I  = skipped (same as input)
             | 
15:38:33.036 I content/docs/page-editing-and-options/index.html
15:38:33.185 I  = skipped (same as input)
             | 
15:38:33.187 I content/docs/getting-started/index.html
15:38:33.311 I  = skipped (same as input)
             | 

[11ty] Copied 1 file / Wrote 13 files in 1.99 seconds (153.1ms each, v2.0.1)
[11ty] Watching…
[Browsersync] Access URLs:
 -----------------------------------
    Local: http://localhost:8080
 External: http://192.168.2.107:8080
 -----------------------------------
[Browsersync] Serving files from: docs
[Browsersync] Watching files...
[Browsersync] Reloading Browsers...
```

**Comment by slaivyn** [comment]
@genemars what I provided is the full terminal output...

I have the same problem with node v18.16.1.

And simply `npx zuix` triggers the same error:
```
npx zuix
WARNING: No configurations found in configuration directory:/home/sylvain/projects/theories/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.
/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/config/lib/config.js:179
    throw new Error('Configuration property "' + property + '" is not defined');
    ^

Error: Configuration property "zuix" is not defined
    at Config.get (/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/config/lib/config.js:179:11)
    at Object.<anonymous> (/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/zuix/commands/generate.js:36:25)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/home/sylvain/.npm/_npx/e326020e254dca45/node_modules/zuix/zuix.js:33:18)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)

Node.js v18.16.1
`̀`

**Comment by genemars** [comment]
I cannot reproduce the issue, this is what I see when I run the command:
```
$ npx zuix

Usage: zuix [options] [command]

Options:
  -h, --help                                       display help for command

Commands:
  version                                          output CLI version
  new [options] <project_name>                     Creates a new project
  start                                            Starts the development server
  generate|g <schematic> [options...]              Generates and/or modifies files based on a schematic
  compile|c <inputFile> [outputFile] [pathPrefix]  Compiles a page
  help [command]                                   display help for command

```

Try running this command:
```
npx zuix@latest
```
it should install the latest version of the *zuix CLI*.

**Comment by slaivyn** [comment]
same result unfortunately...
```
npx zuix@latest
Need to install the following packages:
  zuix@0.4.2
Ok to proceed? (y) y
npm WARN deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm WARN deprecated workbox-cacheable-response@6.6.0: workbox-background-sync@6.6.0
WARNING: No configurations found in configuration directory:/home/sylvain/projects/theories/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.
/home/sylvain/.npm/_npx/e14ad2e6cfe0bd29/node_modules/config/lib/config.js:179
    throw new Error('Configuration property "' + property + '" is not defined');
    ^

Error: Configuration property "zuix" is not defined
    at Config.get (/home/sylvain/.npm/_npx/e14ad2e6cfe0bd29/node_modules/config/lib/config.js:179:11)
    at Object.<anonymous> (/home/sylvain/.npm/_npx/e14ad2e6cfe0bd29/node_modules/zuix/commands/generate.js:36:25)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/home/sylvain/.npm/_npx/e14ad2e6cfe0bd29/node_modules/zuix/zuix.js:33:18)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)

Node.js v18.16.1
```

Do you have any cue about this missing configuration property?

I have a similar result with another computer (with node v14) without the stacktrace so only the message `Configuration property "zuix" is not defined`

**Comment by genemars** [comment]
Hi @slaivyn ,
I just published `zuix@0.4.3`  with the fix for this issue.
Let me know if it works now.

**Comment by slaivyn** [comment]
Hi @genemars,
It works!
Thank you!
